### ISSUE TITLE
throw error in sveltekit project

### ISSUE THREAD

**Issue by Sooraj-s-98** [vercel/satori#185]
# Bug report

## Description / Observed Behavior

nbind.js:1146 Uncaught (in promise) ReferenceError: _a is not defined

## Expected Behavior

no error  

## Reproduction

https://satori.up.railway.app/

<img width="1424" alt="svelte" src="https://user-images.githubusercontent.com/52352285/195385550-a79acb23-797a-4d24-a17e-2bb45aab5f80.png">

**Comment by shuding** [comment]
Can you share your code?

**Comment by Sooraj-s-98** [comment]
@shuding  sure 

https://github.com/Sooraj-s-98/satori-play

**Comment by wong2** [comment]
Related: https://github.com/diegomura/react-pdf/issues/662 https://github.com/vadimdemedes/yoga-layout-prebuilt/issues/2

**Comment by wong2** [comment]
TLDR: the js file generated by nbind(which is used by yoga to build javascript lib) is not usable under strict mode.

**Comment by shuding** [comment]
I think we can workaround it in our bundling script (as we have fixed the versions of yoga-layout-prebuilt and yoga-wasm-web).

**Comment by wong2** [comment]
I thought the build script doesn't bundle `yoga-layout-prebuilt`?

**Comment by shuding** [comment]
Yeah, I think we should be able to bundle it.

**Comment by mattjennings** [comment]
You can workaround this by adding the following to your vite config (or creating a vite.config.js if you don't have one):

```js
// vite.config.js
export default {
    /* other config */

    define: {
       _a: 'undefined'
    }
}
```

**Comment by shivam-rcfs** [comment]
It works but there are more things to look in like flex property on every div element.

**Comment by shuding** [comment]
> flex property on every div element

@shivam-rcfs That is expected. Satori doesn't work like the standard CSS where `div` is a `block` by default. It is required to use `display: flex` everywhere.

**Comment by gfox1984** [comment]
> You can workaround this by adding the following to your vite config (or creating a vite.config.js if you don't have one):
> 
> ```js
> // vite.config.js
> export default {
>     /* other config */
> 
>     define: {
>        _a: 'undefined'
>     }
> }
> ```

It works... in dev environment. But if you build and preview (or deploy) your project, you'll still get this error in the console.

**Comment by gfox1984** [comment]
I've fixed the issue (both in dev and prod) using a patch created with [patch-package](https://github.com/ds300/patch-package).

The patch is called `yoga-layout-prebuilt+1.10.0.patch` and it contains:
```
diff --git a/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js b/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js
index 0853309..5354776 100644
--- a/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js
+++ b/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js
@@ -1143,7 +1143,7 @@
       }
     }_nbind.addMethod = addMethod;function throwError(message) {
       throw new Error(message);
-    }_nbind.throwError = throwError;_nbind.bigEndian = false;_a = _typeModule(_typeModule), _nbind.Type = _a.Type, _nbind.makeType = _a.makeType, _nbind.getComplexType = _a.getComplexType, _nbind.structureList = _a.structureList;var BindType = function (_super) {
+    }_nbind.throwError = throwError;_nbind.bigEndian = false;var _a; _a = _typeModule(_typeModule), _nbind.Type = _a.Type, _nbind.makeType = _a.makeType, _nbind.getComplexType = _a.getComplexType, _nbind.structureList = _a.structureList;var BindType = function (_super) {
       __extends(BindType, _super);function BindType() {
         var _this = _super !== null && _super.apply(this, arguments) || this;_this.heap = HEAPU32;_this.ptrSize = 4;return _this;
       }BindType.prototype.needsWireRead = function (policyTbl) {

```

See the readme in [patch-package](https://github.com/ds300/patch-package) if you're not sure how to use it.

Note: solutions that use a rollup plugin don't work (such as https://github.com/facebook/yoga/issues/798#issuecomment-555858987). It seems that the timing of the replacement is not right (the following statements won't resolve `_a` properly.)

**Comment by TinsFox** [comment]
I have the same issue too in vite + react project.

**Comment by shuding** [comment]
I believe this is now fixed as we replaced the prebuilt yoga completely. Let me know if it is not!

**Comment by Sooraj-s-98** [comment]
not fixed, you can check here https://satori-play-production.up.railway.app/

**Comment by jeetiss** [comment]
Update `satori` to latest version, as i see you still use old version with broken `yoga-layout-prebuilt`
